MedicationScheduler/MedSchedulerApp/templates/front.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+    <div>
+        <img class="img-fluid" src="{% static 'images/main_img.jpg' %}" alt="Main Image" />
+    </div>
```

Comment:
I think we should add a login form to the front page (maybe off to the side). It doesn't have to be in this PR just making a note for the future. I also made a similar comment on the issue for the front page.

Reply:
@bew5294 okay :)